share/spice/github/github.js

```diff
@@ -62,10 +63,10 @@
 }(this));
 
 // Make sure we display only three items.
-Handlebars.registerHelper("GitHub_last_pushed", function(pushed) {
+Handlebars.registerHelper("GitHub_last_pushed", function(pushed_at) {
```

Comment:
@joerussbowman we now have moment.js available to use. If you wrap `Spice.add()` in `DDG.require('moment.js', function() { ... });` you'll have `moment()` available to you which should allow you to get rid of this helper, or at least simplify it.